subsys/bluetooth/mesh/dfu_cli.c

```diff
@@ -148,11 +148,11 @@ static int req_setup(struct bt_mesh_dfu_cli *cli, enum req type, uint16_t addr,
 	return 0;
 }
 
-static int req_wait(struct bt_mesh_dfu_cli *cli, k_timeout_t timeout)
+static int req_wait(struct bt_mesh_dfu_cli *cli, k_timeout_t wait_to)
```

Comment:
Suggest to rename global `timeout` to `dfu_cli_timeout` instead, to follow the pattern you have shown in this PR (i.e. more specific names for global variables, shorter names for function arguments)

Reply:
Renamed the global one.